### PR TITLE
Removed Instances that were -1 meaning they had no values.

### PR DESCRIPTION
**Pull request by kmada97**

### CODE REVIEW

File: sources/pmwebapi.go [hunk 1]
```diff
@@ -98,6 +98,7 @@ func getRequest(url string) []byte {
 	defer getResponse.Body.Close()
 	getResponseBody, err := ioutil.ReadAll(getResponse.Body)
 	if err != nil {
+
```
Comment (roclark) [inline sources/pmwebapi.go]: Probably don't want this extra line here.

**Review by roclark — COMMENTED** [review]
Looks good to me and looks like it worked as expected when you demoed it to me just a bit ago. Once the extra line is deleted and Joe approves, I am good to merge this.

File: sources/pmwebapi.go (hunk 1 above)
Comment (joehandzik) [reply sources/pmwebapi.go]: Agreed, remove this.

File: sources/pmwebapi.go [hunk 2]
```diff
@@ -135,11 +137,18 @@ func newPcpSource() (PcpSource, error) {
 		if err != nil {
 			log.Fatal("NewRequest: ", err)
 		}
+		
 		for _, getValues := range timestampHeaderVar.Values {
 			for _, instanceList := range getValues.Instances {
+				labelNew := []string{"instance"}
```
Comment (joehandzik) [inline sources/pmwebapi.go]: For this, I'd probably rewrite to be more like this:

labelNew := []string{}
labelValuesNew := []string{}

if instanceList.Instance != -1 {
    labelNew = append(labelNew, "instance")
    labelValuesNew = append(labelValuesNew, strconv.FormatInt(InstanceList.Instance, 10))
}

**Review by roclark — APPROVED** [review]
LGTM!

**Pull request merged**